### ISSUE TITLE
How to use it on Windows? 

### ISSUE THREAD

**Issue by jumperrg** [impsnldavid/imp.push#6]
Hi! 

I wanted to try the library, but didn't find any documentation, and releases for Windows. Can you give some advices how to build it for windows, and how the end app should loook like?  Is it M4L device? 

Thanks!

**Comment by pixsperdavid** [comment]
https://github.com/pixsperdavid/imp.push/releases/tag/v0.1.1

The compiled binaries are available on the release page above (the imp.push.zip link), sounds like you're downloading the source. This is an external object for Max, not an M4L device.

**Comment by pixsperdavid** [comment]
This is not an app, it is an object for Max to allow you to display arbitrary video on the Push display. To install you simply copy the 'imp.push' directory from the above zip file to the Max packages directory.

**Comment by jumperrg** [comment]
> https://github.com/pixsperdavid/imp.push/releases/tag/v0.1.1
> 
> The compiled binaries are available on the release page above (the imp.push.zip link), sounds like you're downloading the source. This is an external object for Max, not an M4L device.

Thank you for the answering! 

Yes, I asked about how to compile it from source because at first I made exactly as you described - copied the imp.push directory from the .zip file, but I pasted it into my custom folder, where all my max4live devices are placed.

It's not a standart max packages directory. Is it essential to paste it into standart directory?

And the second question is what should I do next? I see the folders, but there is no any max devices inside it 
![image](https://user-images.githubusercontent.com/20793480/227783773-0a4652ff-3b13-4bb2-9f6f-d4b9c801d4ff.png)

Thanks!
